BlackMarket/Networking/RSAPIError.swift

```diff
@@ -0,0 +1,50 @@
+//
+//  RSAPIError.swift
+//  BlackMarket
+//
+//  Created by Santiago Andracchi on 27/09/2023.
+//
+
+import RSSwiftNetworking
+import Foundation
+
+struct RSAPIError: Codable, Error {
+  public let statusCode: Int
+  public let message: String?
+  public let messages: [String]?
```

Comment:
could this be a non-optional array instead. You could initialize it with an empty array instead of adding a third state to it.